src/common/users.ts

```diff
@@ -101,10 +102,9 @@ export const getUserReadingRank = async (
       'lastWeek',
     )
     .addSelect(
-      `count(*) filter(where "timestamp" at time zone '${timezone}' >= date_trunc('day', ${now}))`,
-      'today',
+      `MAX(date_trunc('second', "timestamp" at time zone '${timezone}'))`,
```

Comment:
Why do you apply `date_trunc` it will mess the also the client code.
We need to return the full date time

Reply:
I was thinking to remove the milliseconds since there are differences between the saved value (6 digits) vs the parsed value on js (3 digits). But after realizing the case, I agree, this shouldn't be necessary.